apps/ideas/templates/meinberlin_ideas/includes/idea_list_item.html

```diff
@@ -27,11 +27,13 @@ <h3 class="list-item__title">
         {{ object.name }}
         </a>
     </h3>
-    {% if object.category %}
-        <div class="list-item__labels">
-            <span class="label label--big">{{ object.category }}</span>
-        </div>
-    {% endif %}
+    <div class="list-item__labels">
+        {% if object.category %}
```

Comment:
Can we wrap the if around the div? Otherwise an empty div might create some unneeded margin.